sycl/doc/design/UnifiedRuntime.md

```diff
@@ -0,0 +1,21 @@
+# Unified Runtime
+
+## Overview
+The Unified Runtime project serves as an interface layer between the DPC++
+runtime and the device-specific runtime layers which control execution on
+devices. The parts of it primarily utilized by DPC++ are its C API, loader
+library, and the adapter libraries that implement the API for various backends.
+
+The DPC++ runtime accesses the UR api via the [Plugin](https://github.com/intel/llvm/blob/sycl/sycl/source/detail/plugin.hpp)
```

Comment:
Will the name `Plugin` be changed in the future?

Reply:
it probably would make sense to divorce it from the old terminology, I'm thinking possible alternatives could be `Backend` or maybe `Adapter` - what do you think?

Reply:
Agree. My understanding is that `Backend` is good alternative for SYCL Runtime plug-ins, whereas `Adapter` is a for Unified Runtime plug-ins.

Reply:
this was quite a noisy change so I spun it out into its own PR https://github.com/intel/llvm/pull/15363